sdk/src/client/node_api/indexer/query_parameters.rs

```diff
@@ -2,14 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
 
 //! Query parameters for output_id requests
+//! https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/tip48/tips/TIP-0048/openapi3-indexer.yaml
```

Comment:
Maybe put this behind some text?

Reply:
What do you mean? Just explaining what this link is ?